ftw/htmlblock/upgrades/20191210181147_provide_precompiled_bundle/registry.xml

```diff
@@ -0,0 +1,18 @@
+<registry>
+
+    <records prefix="plone.resources/htmlblock"
+           interface='Products.CMFPlone.interfaces.IResourceRegistry'
+           remove="True"/>
+
+    <records prefix="plone.bundles/ftw-htmlblock-resources"
+             interface='Products.CMFPlone.interfaces.IBundleRegistry'>
+        <value key="enabled">True</value>
```

Comment:
Thinking about it, not sure if we should try 

```
        <value key="resources" purge="true">
        </value>
```
but I've not done it myself yet. @buchi any thoughts on that?

Reply:
Done